api/provider/local-bkp/main.go

```diff
@@ -172,7 +172,7 @@ func CreateCluster(logging log.Logger, localConfig util.LocalProvider) error {
 	return nil
 }
 
-func (p printer) Printer(logging log.Logger,isHA bool, operation int) {
+func (p printer) Printer(logging log.Logger, isHA bool, operation int) {
```

Comment:
Can we find an alternative to isHA bool thing and also the `operation int` is not a good way to tell Printer either go with enum or something like that for readability or anything else